doc/http.qbk

```diff
@@ -25,20 +25,43 @@ compliant with the Hypertext Transfer Protocol and the supplements that
 follow. Unfortunately reliable implementations or industry standards do not
 exist in C++.
 
-Beast.HTTP is built on Boost.Asio and uses HTTP parser from NodeJS, which is
-extensively field tested and exceptionally robust. A proposal to add networking
-functionality to the C++ standard library, based on Boost.Asio, is under
-consideration by the standards committee. Since the final approved networking
-interface for the C++ standard library will likely closely resemble the current
-interface of Boost.Asio, it is logical for Beast.HTTP to use Boost.Asio as its
-network transport.
+Beast.HTTP is built on Boost.Asio and uses its own robust header-only HTTP/1
+message parser modeled after the nodejs http-parser (written in C). A proposal
+to add networking functionality to the C++ standard library, based on
+Boost.Asio, is under consideration by the standards committee. Since the final
+approved networking interface for the C++ standard library will likely closely
+resemble the current interface of Boost.Asio, it is logical for Beast.HTTP to
+use Boost.Asio as its network transport.
 
-[heading Scope]
+[endsect]
+
+
+
+[section:scope Scope]
+
+This library is designed to be a building block for creating higher level
+libraries. It is not designed to be end-user facing. There is no convenient
+class that implements the core of a web server, nor is there a convenient
+class to quickly perform common operations such as fetching a file or
+connecting and retrieving a document from a secure connection. These
+use-cases are important. But this library does not try to do that. Instead,
```

Comment:
IMHO, this works better as one sentence. "These use-cases are important, but this library does not try to do that."